docs/GettingStarted.md

```diff
@@ -81,21 +81,35 @@ jest --init
 
 ### Using Babel
 
-To use [Babel](http://babeljs.io/), install the `babel-jest` and `@babel/core` packages via `yarn`:
+To use [Babel](http://babeljs.io/), install required development dependencies via `yarn`:
 
 ```bash
-yarn add --dev babel-jest @babel/core
+yarn add --dev @babel/core @babel/preset-env @babel/preset-react
```

Comment:
this removed `babel-jest`, which is the most important module here (beyond 100% standard Babel setup)

Reply:
That was intentional, below this it says:

> `babel-jest` is automatically installed when installing Jest

So is it necessary to have users also install it here? Especially since they won't actually be importing it anywhere.

Reply:
Yes - the only reason it works without installing it explicitly is due to hoisting performed by `yarn`/`npm`.

I'm not sure how to best formulate it - it _is_ installed automatically, and it _is_ used if you do not configure anything. However, relying on that behavior without installing `babel-jest` yourself is brittle and tied to implementation details of package managers